$NetBSD: patch-configure,v 1.14 2016/06/21 12:57:39 tsutsui Exp $

- setuid root is handled by SPECIAL_PERMS in pkgsrc
- fix a bug that skk is always enabled

--- configure.orig	2016-05-08 06:47:04.000000000 +0000
+++ configure
@@ -21909,7 +21909,7 @@ else
 fi
 
 if test "$gui" = "fb" ; then
-	INSTALL_OPT="-m 4755 -o root"
+	INSTALL_OPT=""
 fi
 
 
@@ -23610,7 +23610,11 @@ if test "${enable_skk+set}" = set; then 
   enableval=$enable_skk; skk=$enable_skk
 fi
 
-if test "$enable_shared" != "no" ; then
+if test "$enable_shared" = "no" ; then
+	skk="no"
+fi
+
+if test "x$skk" != "xno" ; then
 	MAKE_DIRS="inputmethod/skk ${MAKE_DIRS}"
 	OUTPUT_FILES="inputmethod/skk/Makefile ${OUTPUT_FILES}"
 	if test "x$with_libltdl" != xno ; then
